v20.1/temporary-tables.md

```diff
@@ -0,0 +1,211 @@
+---
+title: Temporary Tables
+summary: CockroachDB supports session-scoped temporary tables.
+toc: true
+---
+
+<span class="version-tag">New in v20.1:</span> CockroachDB supports session-scoped temporary tables (also called "temp tables"). Unlike [persistent tables](create-table.html), temp tables can only be accessed from the session in which they were created, and they are dropped at the end of the session.
+
+CockroachDB also supports [temporary views](views.html#temporary-views) and [temporary sequences](create-sequence.html#temporary-sequences).
+
+{{site.data.alerts.callout_danger}}
+**This is an experimental feature**. The interface and output are subject to change. For details, see the tracking issue [cockroachdb/cockroach#46260](https://github.com/cockroachdb/cockroach/issues/46260).
+{{site.data.alerts.end}}
+
+{{site.data.alerts.callout_info}}
+By default, temp tables are disabled in CockroachDB. To enable temp tables, set the `experimental_enable_temp_tables` [session variable](set-vars.html) to `on`.
+{{site.data.alerts.end}}
+
+## Details
+
+- Temp tables are automatically dropped at the end of the session.
+- A temp table can only be accessed from the session in which it was created.
+- Temp tables persist across transactions in the same session.
+- Persistent tables cannot reference temp tables, but temp tables can reference persistent tables.
```

Comment:
I think this would be easier to read if the clauses were reversed, since every other sentence starts with "temp tables are X".

E.g., "Temp tables can reference persistent tables, but persistent tables cannot reference temp tables"